IconTest.fromBitmapAdaptive() test fails

```
java.lang.AssertionError: expected:<189> but was:<188>
at org.junit.Assert.fail(Assert.java:88)
at org.junit.Assert.failNotEquals(Assert.java:834)
at org.junit.Assert.assertEquals(Assert.java:118)
at org.junit.Assert.assertEquals(Assert.java:144)
at androidx.graphics.drawable.IconTest.fromBitmapAdaptive(IconTest.kt:50)
at java.lang.reflect.Method.invoke(Native Method)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
at org.junit.runners.Suite.runChild(Suite.java:128)
at org.junit.runners.Suite.runChild(Suite.java:27)
at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
at android.support.test.internal.runner.TestExecutor.execute(TestExecutor.java:58)
at android.support.test.runner.AndroidJUnitRunner.onStart(AndroidJUnitRunner.java:375)
at android.app.Instrumentation$InstrumentationThread.run(Instrumentation.java:2074)
```

--- a/src/androidTest/java/androidx/graphics/drawable/IconTest.kt
+++ b/src/androidTest/java/androidx/graphics/drawable/IconTest.kt
@@ -39,14 +39,14 @@ class IconTest {
     @Test fun fromBitmapAdaptive() {
         val density = context.resources.displayMetrics.density
 
-        val edge = (108 * density).toInt()
+        val edge = (108.0f * density + 0.5f).toInt()
         val bitmap = Bitmap.createBitmap(edge, edge, ARGB_8888).apply {
             eraseColor(Color.RED)
         }
         val icon = bitmap.toAdaptiveIcon()
 
         val rendered = icon.toIntrinsicBitmap()
-        val masked = (72 * density).toInt()
+        val masked = (72.0f * density + 0.5f).toInt()
         assertEquals(masked, rendered.width)
         assertEquals(masked, rendered.height)
         // Grab a pixel from the middle to ensure we are not being masked.

